docs/getting_started.rst

```diff
@@ -94,13 +133,14 @@ We can also convert ``PauliString`` objects back to dense numpy arrays if we'd l
     # Returns "XYZ"
     P_str = str(P)
 
-For more details on the ``PauliString`` class, see the Python or C++ API documentation.
+For more details on the ``PauliString`` class, see the :doc:`python_api` or :doc:`cpp_api` documentation.
 
 Pauli Operators
 ------------------------
 
-The ``PauliOp`` class lets us represent operators that are linear combinations of Pauli strings with complex coefficients. More specifically,
-we can represent an arbitrary operator :math:`A` as a sum of Pauli strings :math:`P_i` with complex coefficients :math:`c_i`:
+The ``PauliOp`` class lets us represent operators that are linear combinations of Pauli strings with complex coefficients.
+In physics, an operator is represented by a matrix in a given basis.
+For example, in the Pauli basis, we can represent an arbitrary operator :math:`A` as a sum of Pauli strings :math:`P_i` with complex coefficients :math:`c_i`:
```

Comment:
In your examples I would make it clear that the coeff array is c_i and the pauli string array is P_i to link it back with the math.